src/libraries/Common/tests/System/Net/Sockets/SocketTestExtensions.cs

```diff
@@ -38,14 +38,21 @@ public static void ForceNonBlocking(this Socket socket, bool force)
             }
         }
 
-        public static (Socket, Socket) CreateConnectedSocketPair()
+        public static (Socket client, Socket server) CreateConnectedSocketPair() => CreateConnectedSocketPair(IPAddress.Loopback, false);
+
+        public static (Socket client, Socket server) CreateConnectedSocketPair(IPAddress serverAddress, bool dualModeClient)
```

Comment:
Nit: It doesn't make any sense to pass in anything other than either Loopback or IPv6Loopback here, right? So I'd suggest just changing this to "bool ipv6" or similar, just to make this clearer.

Reply:
Also, may as well use default arguments here.

Reply:
fixed